djmoney/models/fields.py

```diff
@@ -147,22 +147,20 @@ def __use_l10n(self):
         return self.use_l10n
 
     def __unicode__(self):
-
-        if self.__use_l10n():
-            locale = self.__get_current_locale()
-            if locale:
-                return format_money(self, locale=locale)
-
-        return format_money(self)
+        return self.__str__()
 
     def __str__(self):
+        try:
+            decimal_places = self.decimal_places
```

Comment:
You could do `getattr(self, 'decimal_places', 2)` here instead of the whole try-except